include/service/entities/site_query_service.js

```diff
@@ -260,13 +260,18 @@ module.exports = function SiteQueryServiceModule(pb) {
    * @param {Function} cb
    */
   SiteQueryService.prototype.getCollections = function (cb) {
-    this.listCollections({ name: { $nin: ["system.indexes", "system.namespaces"] }}, function(err, items) {
-      if(pb.util.isError(err)) {
-        pb.log.error(err);
-      }
-      cb(err, items);
-    });
-  };
+      this.listCollections({}, function(err, items) {
+          if(pb.util.isError(err)) {
+            pb.log.error(err);
+          }
+
+          items = items.filter(function(item) {
```

Comment:
One more small change and I am good to merge.  If there is an error it will be logged.  However, the ```items``` parameter will most likely be null or undefined.  In the ```if``` statement above we should callback with the error and return otherwise we may run into a null reference error here on line 268 when we attempt to filter the array.